pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1528,3 +1524,11 @@ func cidrStrToIPNet(cidr string) (*controlplane.IPNet, error) {
 	}
 	return ipNet, nil
 }
+
+// internalNetworkPolicyKeyFunc knows how to make key for internal NetworkPolicy based on the object metadata of the
+// original NetworkPolicy.
```

Comment:
maybe: "internalNetworkPolicyKeyFunc knows how to generate the key for an internal NetworkPolicy based on the object metadata of the corresponding original NetworkPolicy resource (also referred to as the "source")"

Reply:
Done, thanks.